### PR TITLE
Inferred date CSV exports (#1399) and rationale field (#1401)

### PR DESCRIPTION
**Pull request by blms**
## In this PR

Per #1401:
- Add "rationale" choices field to document inferred datings

Per #1399:
- Include inferred dating fields in document CSV exports

### CODE REVIEW

File: geniza/corpus/migrations/0041_dating_rationale.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        migrations.AddField(
+            model_name="dating",
+            name="rationale",
+            field=models.CharField(
+                choices=[
+                    ("PA", "Paleography"),
+                    ("PE", "Person mentioned"),
+                    ("E", "Event mentioned"),
+                    ("C", "Coinage"),
+                    ("O", "Other (please specify)"),
```
Comment (rlskoeser) [inline geniza/corpus/migrations/0041_dating_rationale.py:22]: so interesting! cool to see these options spelled out

File: geniza/corpus/metadata_export.py [hunk 2]
```diff
@@ -74,6 +78,7 @@ def get_queryset(self):
                 "secondary_languages",
                 "log_entries",
                 "log_entries__user",
+                "dating_set",
```
Comment (rlskoeser) [inline geniza/corpus/metadata_export.py:81]: Nice. Were you able to confirm the export is able to prefetch all the related data and doesn't have to make any new queries per item?

**Review by rlskoeser — APPROVED** [review]
LGTM. Will be great to have these fields included in the export

File: geniza/corpus/metadata_export.py (hunk 2 above)
Comment (blms) [reply geniza/corpus/metadata_export.py:81]: Confirmed! 👍

**Pull request merged**